_posts/2023-06-15-Client-cert-auth-without-roles.adoc

```diff
@@ -0,0 +1,241 @@
+```
+
+And finally, we need to configure the undertow subsystem to make use of our `http-authentication-factory`. Please run the following commands
+in a batch as specified below to avoid unnecessary warnings: 
+```
+batch
+#configure the undertow subsystem	
+/subsystem=undertow/application-security-domain=other:undefine-attribute(name=security-domain)
+/subsystem=undertow/application-security-domain=other:write-attribute(name=http-authentication-factory,value=clientCertAuth)
+/subsystem=undertow/application-security-domain=other:write-attribute(name=override-deployment-config,value=true)
+run-batch
+```
+
+And to apply all these changes, please type in `reload` into the same terminal. Now that we have configured the server, we are ready to deploy and test our application.
+
+== Deploying and accessing the application
+  
+We’re going to make use of the ```simple-webapp``` project. It can be deployed using the following commands:
+
+```
+cd $PATH_TO_ELYTRON_EXAMPLES/client-cert-auth-without-roles/simple-webapp
+
+mvn clean install wildfly:deploy
+```
+
+*Note:* There is also a repository called `elytron-examples/simple-webapp`. But we will not be using that. Instead, we will be deploying a slightly modified version of that web application located inside this repository.
+
+== Import the Certificate into Your Browser
+
+Before you access the application, you must import the _client1.keystore.pkcs12_, which holds the client certificate, into your browser.
+
+== Import the Certificate into Google Chrome
+
+ - Click the Chrome menu icon (3 dots) in the upper right on the browser toolbar and choose *Settings*. This takes you to `link:`chrome://settings/`.
+
+- Click on *Privacy and security* and then on *Security*.
+
+- Scroll down to the *Advanced* section and on the *Manage certificates* screen, select the *Your Certificates* tab and click on the *Import* button.
+
+- Select the *client1.keystore.pkcs12* file. You will be prompted to enter the password: `keystorepass`.
+
+- The client certificate is now installed in the Google Chrome browser.
+
+You can follow the same instructions to import `client2.keystore.pkcs12` into your browser. 
+
+== Import the Certificate into Mozilla Firefox
+
+- Click the *Edit* menu item on the browser menu and choose *Settings*.
+
+- A new window will open. Click on *Privacy & Security* and scroll down to the *Certificates* section.
+
+- Click the *View Certificates* button.
+
+- A new window will open. Select the *Your Certificates* tab and click the *Import* button.
+
+- Select the *client1.keystore.pkcs12* file. You will be prompted to enter the password: `keystorepass`.
+
+- The certificate is now installed in the Mozilla Firefox browser.
+
+You can follow the same instructions to import `client2.keystore.pkcs12` into your browser. Now try accessing the application using https://localhost:8443/simple-webapp.
```

Comment:
"Now try accessing..." could be moved to a new line and the section could be called `== Access the Application`.